Azure/azure-linux/ingest/parser.yml

```diff
@@ -0,0 +1,282 @@
+      name: kv.parse-kv
+      properties:
+        input_field: '{{ grok_omsagent.result.omsagent_message }}'
+        output_field: result
+        value_sep: '='
+        item_sep: \s;\s
+  - name: grok2_omsagent
+    description: ''
+    filter: '{{ grok_omsagent.result.omsagent_message != null }}'
+    external:
+      name: grok.match
+      properties:
+        input_field: '{{omsagent.result.COMMAND }}'
+        output_field: result
+        pattern: '%{UNIXPATH:process_executable}( %{GREEDYDATA:process_args})?'
+        custom_patterns: {}
+  - name: omsagent2
+    description: ''
+    filter: '{{ grok_omsagent.result.omsagent_message != null }}'
+    external: null
+  - name: action-outcome
+    description: ''
+    filter: >-
+      {{json_event.message.properties.ident == "systemd" or
+      json_event.message.properties.ident == "sudo"}}
+    external: null
+stages:
+  stage2:
+    actions:
+      - set:
+          log.level: '{{json_event.message.level}}'
+          process.pid: '{{json_event.message.properties.pid}}'
+          process.id: '{{json_event.message.properties.pid}}'
+          log.hostname: '{{json_event.message.properties.hostname}}'
+          event.created: '{{json_event.message.time}}'
+          event.message: '{{json_event.message.properties.Msg}}'
+        filter: ''
+        name: set
+      - set:
+          os.family: linux
+          os.platform: linux
+        filter: ''
+        name: set
+      - dictionary:
+          err: error
+          crit: critical
+          info: info
+          warn: warn
+          error: error
+          notice: info
+          critical: critical
+        mapping:
+          log.level: log.level
+        filter: ''
+        name: translate
+  sshd:
+    actions:
+      - set:
+          source.ip: '{{ grok_sshd.result.source_ip }}'
+          user.name: '{{ grok_sshd.result.user_name }}'
+          source.port: '{{ grok_sshd.result.source_port }}'
+          source.domain: '{{ grok_sshd.result.source_domain }}'
+        filter: ''
+        name: set
+  systemd:
+    actions:
+      - set:
+          user.name: '{{ grok_systemd.result.user_name }}'
+          action.type: '{{ grok_systemd.result.systemd_observation }}'
+        filter: ''
+        name: set
+      - set:
+          action.name: systemd:session
+          action.type: open
+        filter: '{{ grok_systemd.result.systemd_observation == "Started Session" }}'
+        name: set
+      - set:
+          action.target: user
+        filter: '{{ grok_systemd.result.user_name != null }}'
+        name: set
+  crond:
+    actions:
+      - set:
+          user.name: '{{ grok_crond.result.user_name }}'
+          process.args: '{{ grok_crond.result.process_args.split() }}'
+          process.executable: '{{ grok_crond.result.process_executable }}'
+        filter: ''
+        name: set
+      - set:
+          process.executable: /bin/sh
+        filter: '{{ grok_crond.result.process_executable == none }}'
```

Comment:
Isn't it hazardous to infer the executable name here?

Reply:
It was done in the original parser, but I can remove it

Reply:
Considering that the process is `/bin/sh` is probably not true in all situations.

IMHO, you’d better put the whole command line to the `process.command_line` and not try to infer either `process.args` and `process.executable`.

Reply:
Changed